src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -1445,6 +1406,108 @@ def _fill_event_detection_kwargs(
 
         return kwargs
 
+    def _init_columns(
+            self,
+            auto_column_detect: bool = False,
+            trial_columns: str | list[str] | None = None,
+            time_column: str | None = None,
+            time_unit: str | None = None,
```

Comment:
if we use `auto_column_detect` as the fourth argument, the arguments would be initializedhow they appear in the function.

Reply:
I would rather like to conform it to the order in `__init__()`